tests/test_model_loading.py

```diff
@@ -108,7 +107,7 @@ def fake_aggregate_task(trunk_model):
     model.key = AssetKeys.valid_head_model
 
     aggregate_task = Mock(spec=substra.models.Task)
-    aggregate_task.rank = 3
+    aggregate_task.rank = 1
```

Comment:
or reduced TO one? 🤔

Reply:
It is for the "multiple task error", in order to be able to now which round to ask for. Rounds were defined arbitrary. And if we put  None, it does not work as we take the last ended task only.